website/static/js/registrationUtils.js

```diff
@@ -247,10 +247,18 @@ var MetaSchema = function(params) {
     self.schema = params.schema || {};
     self.id = [self.name, self.version].join('_');
 
+    self.requiresApproval = params.requires_approval || false;
+    self.fulfills = params.fulfills || [];
+
+    var count = 1;
     $.each(self.schema.pages, function(i, page) {
         var mapped = {};
         $.each(page.questions, function(qid, question) {
-            mapped[qid]  = new Question(question, qid);
+            // This is necessary now since page.questions is a list
```

Comment:
@barbour-em TY for the docstrings. I like this solution.

Can you please update all of the other version 2 schemas to use this approach?